colcon_bundle/installer/assets/gazebo_blacklist.txt

```diff
@@ -375,4 +375,8 @@ xdg-user-dirs
 xkb-data
 xml-core
 zip
-zlib1g
\ No newline at end of file
+zlib1g
```

Comment:
I think we can remove this file? I don't think it's used anywhere.

Reply:
Sure, I have removed it.